main/opcodes/stack-operations.zkasm

```diff
@@ -7,6 +7,15 @@
  *  - stack input: none
  *  - stack output: [pushed_value]
  */
+
+opPUSH0:
```

Comment:
Does it not consume any ZK counters? I would think that gas check at stack check do. Even POP requires 100 steps.

Reply:
Yes, we would add them